keyboards/keycult/keycult60/keyboard.json

```
@@ -0,0 +1,603 @@
+{
+    "keyboard_name": "Keycult 60",
+    "manufacturer": "Yiancar-Designs",
+    "url": "https://yiancar-designs.com",
+    "maintainer": "Yiancar-Designs",
+    "usb": {
+        "vid": "0x8968",
+        "pid": "0x6339",
+        "device_version": "0.0.1"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "B6",
+        "on_state": 0
+    },
+    "matrix_pins": {
+        "cols": ["A1", "B1", "B0", "A7", "A6", "A5", "A4", "A3", "B2", "B10", "B11", "B12", "B13", "B14"],
+        "rows": ["A2", "B9", "B8", "B5", "B4"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "STM32F072",
+    "bootloader": "stm32-dfu",
+    "community_layouts": [
+        "60_ansi",
+        "60_ansi_split_bs_rshift",
+        "60_ansi_tsangan",
+        "60_ansi_tsangan_split_bs_rshift",
```

based on novelkeys pre-order information
```suggestion
        "60_ansi_tsangan_split_bs_rshift",
        "60_hhkb",
```